web/crux-ui/e2e/utils/global-setup.ts

```diff
@@ -39,4 +39,4 @@ const globalSetup = async (config: FullConfig) => {
   await browser.close()
 }
 
-export default globalSetup
+export default globalSetup
```

Comment:
don't remove empty lines please. They are great friends of @polaroi8d :)

Reply:
Was not intentional. Thanks!